schema/src/main/jade-tables/alignment_file.table.json

```diff
@@ -11,11 +11,6 @@
       "datatype": "string",
       "type": "repeated"
     },
-    {
-      "name": "byte_size",
```

Comment:
Removed the size column across the board since it's tracked for free by GCS (and doesn't seem like a high-level search criteria, though I could be wrong)